Change streaming terminology to parent/child in docs (#9312)
netdata/netdata: web/server/README.md

### Enabling TLS support

> Since v1.16.0, Netdata supports encrypted HTTP connections to the web server, plus encryption of streaming data between a parent and its child nodes, via the TLS protocol.

Reviewer: ```suggestion
Since v1.16.0, Netdata supports encrypted HTTP connections to the web server, plus encryption of streaming data to a parent from its child nodes, via the TLS protocol.
```
